src/lexer_a.c

```diff
@@ -0,0 +1,58 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        ::::::::            */
+/*                                             :+:    :+:            */
+/*                                                     +:+                    */
+/*   By: mbp14 <mbp14@student.codam.nl>               +#+                     */
+/*                                                   +#+                      */
+/*   Created: 2023/06/25 13:59:24 by mbp14         #+#    #+#                 */
+/*   Updated: 2023/06/25 14:00:17 by mbp14         ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "../include/lexer.h"
```

Comment:
I suggest to do `#include "lexer.h"` instead of `#include "../include/lexer.h"` and set up include folders in Makefile

Reply:
done